schemas/twitter.js

```diff
@@ -0,0 +1,167 @@
+import * as twitter from '../apis/twitter';
+
+import {
+    GraphQLSchema,
```

Comment:
another style nit - the project generally uses 2 space tabs instead of 4, can we update that?

(basically these style things aim to improve consistency and readability, and in the near future ESLint should enforce them)